packages/plans-grid-next/.storybook/main.js

```diff
@@ -1,3 +1,10 @@
 const path = require( 'path' );
+
 const storybookDefaultConfig = require( '@automattic/calypso-storybook' );
-module.exports = storybookDefaultConfig();
+
+module.exports = storybookDefaultConfig( {
+	staticDirs: [ '../static' ],
+	webpackAliases: {
+		[ 'wpcom-proxy-request' ]: require.resolve( '../src/__mocks__/wpcom-proxy-request.js' ),
+	},
```

Comment:
Mocking `wpcom-proxy-request` for the time being due to an issue I encountered with this library and [MSW](https://mswjs.io/). The plan is to remove this mock in a future update, once the underlying issue is resolved.

Reply:
Could you briefly explain the issue? Just cuirous to learn. It'd also be great to add more detailed context for posterity :)